tests/tck/features/go/GO.feature

```diff
@@ -1609,12 +1609,11 @@ Feature: Go Sentence
   Scenario: GroupBy and Count
     When executing query:
       """
-      GO 1 TO 2 STEPS FROM "Tim Duncan" OVER like WHERE like._dst != "YAO MING" YIELD like._dst AS vid
-      | GROUP BY $-.vid YIELD 1 AS id | GROUP BY $-.id YIELD COUNT($-.id);
+      GO 1 TO 2 STEPS FROM "Tim Duncan" OVER like WHERE like._dst != "YAO MING" YIELD like._dst AS vid | GROUP BY $-.vid YIELD $-.vid, 1 AS id | GROUP BY $-.id YIELD $-.id, COUNT($-.id);
       """
     Then the result should be, in any order, with relax comparison:
-      | COUNT($-.id) |
-      | 4            |
+      | $-.id | COUNT($-.id) |
```

Comment:
`GROUP BY $-.id YIELD  COUNT($-.id);` it the the correct representation. why need add a col `$-.id`

Reply:
We support yield sentence that only has aggregate columns, but the items of group by must appear in the yield list.
Similar to the discussion here: `https://github.com/vesoft-inc/nebula-graph/pull/503#discussion_r553698798`

Reply:
`GROUP BY $-.id YIELD COUNT($-.id);` 
the group item `$-.id ` is in yield clause, the `COUNT($-.id);`  means include it.

Reply:
Already fixed. Need not col $-.id.